### PR TITLE
Restore hostname back to 'localhost' in case it was changed

### PR DESCRIPTION
**Pull request by Lex-2008**
...by something else.

Changelog: none

### CODE REVIEW

File: tests/acceptance/04_examples/outputs/check_outputs.cf [hunk 1]
```diff
@@ -22,6 +22,12 @@ bundle agent init
   reports:
     DEBUG_LOTS::
       "$(this.bundle): will consider example $(all_examples)";
+
+  commands:
+      # restore hostname to be "localhost" - it's set to be like this in
+      # https://github.com/mendersoftware/mender-qa/blob/master/scripts/initialize-user-data.sh#L57
+      # but might be changed somewhere else
+      "hostname localhost";
```
Comment (vpodzime) [inline tests/acceptance/04_examples/outputs/check_outputs.cf:30]: Is this test marked as unsafe or is it going to change hostname of my machine the next time I run `./testall` or `make check`?

Comment (Lex-2008) [reply tests/acceptance/04_examples/outputs/check_outputs.cf:30]: good point! Yes, it will change your hostname.

Comment (vpodzime) [reply tests/acceptance/04_examples/outputs/check_outputs.cf:30]: Then it's not a good way to go. (Unless the *check_outputs.cf* test is skipped by default)

Comment (olehermanse) [reply tests/acceptance/04_examples/outputs/check_outputs.cf:30]: Seems like it should not be a part of this test at all? Restore the hostname after the test which changes it finishes.

**Comment by Lex-2008** [comment]
FU at https://github.com/cfengine/buildscripts/pull/843

**Pull request closed without merging**